db-init/src/main/resources/database/migrations/V1.1__BIE_Contention_Event.sql

```diff
@@ -0,0 +1,11 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+CREATE TABLE IF NOT EXISTS "bie_contention_event" (
+    id uuid DEFAULT uuid_generate_v4 () NOT NULL,
+    event VARCHAR NOT NULL,
+    event_details VARCHAR NOT NULL,
```

Comment:
I understand `event_details` is more general and we can still keep it, but I was expecting explicit columns for certain event details.
From https://github.com/department-of-veterans-affairs/abd-vro/issues/1642:
> Persist the events published to these streams somewhere that is queryable. We are interested in the following data elements for each event:
> 
> The event type, e.g. “Contention created”, “Contention updated”, etc.
> The event timestamp
> Benefit Claim ID
> Contention ID
> Diagnostic Code(s)
> Classification (if available)

I was expecting explicit columns for event type, IDs, and especially Diagnostic Code and Classification since Team CC will likely want to query the DB using Diagnostic Code and/or Classification. Also, those columns would be indexed for efficient lookups.
Thoughts?

Reply:
I was considering that to be part of #1680